Python.gitignore

```diff
@@ -94,6 +94,9 @@ venv.bak/
 .spyderproject
 .spyproject
 
+# PyCharm
```

Comment:
.idea folder is not only applicable to PyCharm, but also Rider and possibly other IntelliJ IDEs. I think it would be better to replace the comment `#PyCharm` with `#IntelliJ project specific settings` in the description.